keymap:
  # set the index of the topic[/joy] array.
  axes:
    L_stick_H:      0
    L_stick_V:      1
    l2_triger:      2
    R_stick_H:      3
    R_stick_V:      4
    r2_triger:      5
    cross_key_H:    6
    cross_key_V:    7
  buttons:
    A_button:       0
    B_button:       1
    Y_button:       2
    X_button:       3
    L1_button:      4
    R1_button:      5
    L2_button:      6
    R2_button:      7
    option_button:  8
    start_button:   9
    home_button:    10
    L3_button:      11
    R3_button:      12